azureLogIngestion/adapter/index.ts

```diff
@@ -23,15 +24,15 @@ interface Records {
 
 export default class Adapter {
     eventProcessor: EventProcessor
-    spanProcessor: SpanProcessor
     logProcessor: LogProcessor
     metricsProcessor: MetricsProcessor
+    otelAdapter: OpenTelemetryAdapter
 
     constructor(apiKey: string) {
         this.eventProcessor = new EventProcessor(apiKey)
-        this.spanProcessor = new SpanProcessor(apiKey)
         this.logProcessor = new LogProcessor(apiKey)
         this.metricsProcessor = new MetricsProcessor(apiKey)
+        this.otelAdapter = new OpenTelemetryAdapter(apiKey)
```

Comment:
I intended this adapter to be swappable with the existing adapter or to run along side it, not to be nested within it.

Reply:
@kolanos That'd make more sense. Currently this and the otel branch are nested.